### PR TITLE
Adding Bradford Boyles pyglpk

### PR DESCRIPTION
**Pull request by linsalrob**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pyglpk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: http://tfinley.net/software/pyglpk/
+  license: GPL-3.0-only
+  license_family: GPL
+  license_file: LICENSE.txt
```
Comment (mwcraig) [inline recipes/pyglpk/meta.yaml:42]: Please change this to `LICENSE` to match the name of the license file in the source distribution.

File: recipes/pyglpk/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,621 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (mwcraig) [inline recipes/pyglpk/LICENSE.txt:1]: I think you can remove the license file from the recipe because it is included in the PyPI source distribution.

**Review by mwcraig — CHANGES_REQUESTED** [review]
Thanks for opening this @linsalrob! It looks like it just needs one change. The license file is included in the source distribution so you can remove it from the recipe if you make the change I indicated in the `license_file` field of the recipe.

File: recipes/pyglpk/LICENSE.txt (hunk 2 above)
Comment (linsalrob) [reply recipes/pyglpk/LICENSE.txt]: Committed those changes,  removing the license file and pointing it to the one in the source. Thanks!

**Review by mwcraig — APPROVED** [review]
Thanks @linsalrob -- also just noticed this is your first contribution, so welcome to the community! It will take roughly an hour from when this is merged until you will be able to install it from the conda-forge channel.

**Pull request merged**